recipes/default.rb

```diff
@@ -47,6 +47,12 @@
   cwd "#{node['timesync']['application_path']}/source"
 end
 
+bash 'create root user' do
+  code "npm run create-user -- -u root -p #{secret_key['root_pass']}"
+  env environment
+  cwd "#{node['timesync']['application_path']}/source"
+end
```

Comment:
I recommend you switch this to using the [execute](https://docs.chef.io/resource_execute.html) resource instead since you're doing a one-liner. This also allows us to use the ``sensitive`` property which hides the secret from being send to the output of the chef run. Can you please switch this to ``execute`` and enable the ``sensitive`` property?